pkg/agent/cniserver/server_windows_test.go

```diff
@@ -49,6 +50,8 @@ import (
 var (
 	containerMACStr = "23:34:56:23:22:45"
 	dnsSearches     = []string{"a.b.c.d"}
+
+	mockWinnet *winnettest.MockInterface
```

Comment:
unit test mocks should never be global variables IMO
they are very much specific to individual tests

Reply:
You are right, but other mock tests (like `mockRoute`, `mockOFClient`) are also used like this in the unit tests of this package and they are initialized in function `newMockCNIServer` in every individual test.